src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.ImmGetContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Win32.Globalization;
+
+namespace Windows.Win32
+{
+    internal static partial class PInvoke
+    {
+        public static HIMC ImmGetContext(IHandle<HWND> hWnd)
+        {
+            HIMC result = ImmGetContext(hWnd.Handle);
+            GC.KeepAlive(hWnd);
```

Comment:
We want to use the following existing pattern: 
```suggestion
        public static HIMC ImmGetContext(in T hWnd) where T : IHandle<HWND>
        {
            HIMC result = ImmGetContext(hWnd.Handle);
            GC.KeepAlive(hWnd.Wrapper);
```